### PR TITLE
task 4 GET /api//articles/:article_id

### PR DESCRIPTION
**Pull request by GNicholson95**

### CODE REVIEW

File: models/model.js [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
-	}).catch((err) => {
-        return err
-      })
+	})
 };
 
+exports.selectArticleById = (Article_id) => {
+	return db.query('SELECT * FROM articles WHERE Article_id = $1;', [Article_id])
+	  .then((result) => {
+		console.log(result.rows);
```
Comment (Beth-McCormack) [inline models/model.js:13]: remove console.logs before committing

File: __tests__/app.test.js [hunk 2]
```diff
@@ -47,4 +47,32 @@ describe('GET /api/topics', () => {
...
+  it('GET:200 and should respond with an object containing correct article properties', () => {
+    return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then((response) => {
+        const { article } = response.body;
+        expect(typeof article).toBe('object')
+        expect (article.hasOwnProperty('author')).toBe(true);
+        expect (article.hasOwnProperty('title')).toBe(true);
+        expect (article.hasOwnProperty('article_id')).toBe(true);
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:61]: You can be abit more specific here - what number do you expect the article id to be?

File: __tests__/app.test.js [hunk 3]
```diff
@@ -47,4 +47,32 @@ describe('GET /api/topics', () => {
...
+        expect (article.hasOwnProperty('body')).toBe(true);
+        expect (article.hasOwnProperty('topic')).toBe(true);
+        expect (article.hasOwnProperty('created_at')).toBe(true);
+        expect (article.hasOwnProperty('votes')).toBe(true);
+        expect (article.hasOwnProperty('article_img_url')).toBe(true);
+      });
+  });
+  it('should throw 404 error if given id doesnt exist', () => {
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:69]: Nice test for a 404! What if the path was /api/articles/banana

**Review by Beth-McCormack — COMMENTED** [review]
Nice work so far, I've left a couple of comments for you

**Comment by GNicholson95** [comment]
Thanks Beth, can I merge this or do I need to resolve these issues first then push again?

**Pull request merged**